app/uk.gov.hmrc.helptosaveproxy/config/AppConfig.scala

```diff
@@ -40,4 +40,7 @@ object AppConfig extends ServicesConfig {
 
   val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}/nsi-services/account"
 
+  def dwpUCClaimantCheckUrl(nino: String): String =
+    s"${baseUrl("dwp")}/v1/isEligible/$nino?systemId=607&thresholdAmount=650&transactionId=2a07d761-70ea-4a10-ae00-00d4dd53dea9"
```

Comment:
the query parameters should be read from config. The transaction ID should be unique as well. Perhaps this should be:
```scala
def dwpUCClaimantCheckUrl(nino: String, transactionId: UUID): String = ...
```
and you'd call it like this:
```scala
dwpUCClaimantCheckUrl("nino", UUID.randomUUID())
```

Reply:
(unique per call to DWP)

Reply:
Ok thanks, I've updated that method and where its used.